sensor_msgs/include/sensor_msgs/distortion_models.hpp

```diff
@@ -31,6 +31,9 @@
 // ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
+// This file is originally ported from ROS1:
```

Comment:
True, but the original file is from a ROS1 repository.  Should I change it to just:

"This file is originally from:"

?